packages/react/src/components/header/Header.module.scss

```diff
@@ -28,10 +31,19 @@
   --action-bar-button-size: 64px;
   --action-bar-item-gutter: 12px;
   --action-bar-item-title-font-size: 14px;
-  --action-bar-item-title-font-color: var(--color-black-90);
+  --action-bar-item-title-font-color: var(--header-color);
+  --lang-selector-item-font-color: var(--header-color);
+  --lang-selector-dropdown-background-color: var(--color-white);
+  --nav-link-font-color: var(--header-color);
+  --nav-border-color: var(--color-black-20);
+  --nav-link-hover-color: var(--color-black-90);
+  --nav-link-dropdown-background-color: var(--color-white);
+  --nav-button-background-color: var(--color-black-5);
+  --nav-button-hover-background-color: var(--color-black-5);
+  --nav-search-color: var(--header-color);
+  --universal-bar-background-color: var(--color-white);
```

Comment:
Universal bar's background color is actually `var(--color-black-5)`.

Reply:
All in all, there's a lot of new colour variables here but I think they're all reasonable and have their place 👍 And I think they'll serve future development as well when we add support for custom theming the Header.